build.gradle

```diff
@@ -4,11 +4,12 @@ buildscript {
     repositories {
         jcenter()
         mavenLocal()
+        maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.2'
+        classpath 'com.android.tools.build:gradle:2.1.3'
         classpath 'com.google.gms:google-services:3.0.0'
-
+        classpath 'io.fabric.tools:gradle:1.+'
```

Comment:
Twitter actually recommends using the open-ended approach: https://fabric.io/kits/android/twitterkit/install 

They don't publish the latest version number anywhere, but some guess and check determined the latest is 1.21.7 so I've changed it to that.